src/server/configs/common.ts

```diff
@@ -36,6 +37,7 @@ export default {
         SuperuserHeader.XDlAllowSuperuser,
         SuperuserHeader.XDlSudo,
         AuthHeader.Authorization,
+        SERVICE_USER_ACCESS_TOKEN_HEADER,
```

Comment:
Let's add it to appSensitiveKeys, appSensitiveHeaders

Reply:
ook, added